run.sh

```diff
@@ -1,12 +1,8 @@
 #!/bin/bash
 set -e
 
-## commented out for now until further work on lxc
-#/patchcontainer.sh
-#brctl addbr lxcbr0
-#ifconfig lxcbr0 10.0.4.1 up
+# This script is run if the jujubox image is run without any arguments.
 
 export HOME=/home/ubuntu
 cd $HOME
-chown ubuntu:ubuntu $HOME/.juju
 sudo -u ubuntu ssh-agent /bin/bash
```

Comment:
I think everything but this line can basically be removed with some tweaks to the dockerfile

```
USER ubuntu
WORKDIR /home/ubuntu
ENV HOME=/home/ubuntu
```

Then you just need to fire up the ssh-agent with /bin/bash.  Pure Vida